Beams/Logic/vtkSlicerBeamsModuleLogic.h

```diff
@@ -80,6 +105,9 @@ class VTK_SLICER_BEAMS_LOGIC_EXPORT vtkSlicerBeamsModuleLogic :
   void operator=(const vtkSlicerBeamsModuleLogic&) = delete;
   
   vtkSlicerMLCPositionLogic* MLCPositionLogic;
+
+private:
+  vtkSmartPointer<vtkSlicerIECTransformLogic> IecLogic = vtkSmartPointer<vtkSlicerIECTransformLogic>::New();

```

Comment:
Don't use `vtkSmartPointer`, use `vtkNew` (you even have it included although it seems not to be used). See examples in Slicer source code (search for .h files containing "vtkNew")